extensions/aris/src/bigDataCluster/commands.ts

```diff
@@ -0,0 +1,76 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as nls from 'vscode-nls';
+import { AppContext } from '../appContext';
+import { TreeNode } from './tree/treeNode';
+import { AddControllerDialog } from './dialog/addControllerDialog';
+import { ControllerTreeDataProvider } from './tree/controllerTreeDataProvider';
+import { ControllerNode, ControllerTreeNode } from './tree/controllerTreeNode';
+import { IEndPoint } from './controller/types';
+
+const localize = nls.loadMessageBundle();
+
+export function registerCommands(appContext: AppContext, treeDataProvider: ControllerTreeDataProvider): void {
```

Comment:
Please remove and just put the 3 command definitions in extension.ts - see comments below on the approach

Reply:
fixed. The pattern was from CMS.